presto-clp/src/main/java/com/facebook/presto/plugin/clp/ClpFilterToKqlConverter.java

```diff
@@ -114,11 +118,39 @@ public ClpExpression visitCall(CallExpression node, Void context)
             if (operatorType.isComparisonOperator() && operatorType != IS_DISTINCT_FROM) {
                 return handleLogicalBinary(operatorType, node);
             }
+            else if (OperatorType.BETWEEN == operatorType) {
+                return handleBetween(operatorType, node);
+            }
         }
 
         return new ClpExpression(node);
     }
 
+    private ClpExpression handleBetween(OperatorType operator, CallExpression node)
+    {
+        if (node.getArguments().size() != 3) {
+            throw new PrestoException(CLP_PUSHDOWN_UNSUPPORTED_EXPRESSION,
+                    "Logical binary operator must have exactly two arguments. Received: " + node);
+        }
+        Optional<String> variableReferenceDefinition = node.getArguments().get(0).accept(this, null).getPushDownExpression();
+        Optional<String> lowerBoundConstantDefinition = node.getArguments().get(1).accept(this, null).getPushDownExpression();
+        Optional<String> upperBoundConstantDefinition = node.getArguments().get(2).accept(this, null).getPushDownExpression();
```

Comment:
Since we rename `definition` to `pushDownExpresson`, we should also rename them.